cli/mender-artifact/main.go

```diff
@@ -478,12 +478,6 @@ func getCliContext() *cli.App {
 		Action: Copy,
 	}
 
-	copy.Flags = []cli.Flag{
-		compressionFlag,
-		privateKeyFlag,
-	}
```

Comment:
Hmm, this looks a little too much to me now, looking at it.
I guess at least the `privateKeyFlag` should stay? And possibly the `compressionFlag` is needed too, in order to actually print the warning when it is set?

Reply:
Well, for now, the `compression` flag does not apply when copying into the Artifact. If compression is set, a warning is issued though, that you should use `mender-artifact modify` to change the compression of your `Artifact`.

Reply:
Ok, sounds good!

Discussed on Slack: Keep both flags, but warn on use of `compression`.